cli/internal/cmd/verify.go

```diff
@@ -459,12 +459,12 @@ func (f *attestationDocFormatterImpl) parseSNPReport(b *strings.Builder, reportB
 	writeIndentfln(b, 2, "Platform Info:")
 	writeIndentfln(b, 3, "Symmetric Multithreading enabled (SMT): %t", platformInfo.SMTEnabled)
 	writeIndentfln(b, 3, "Transparent secure memory encryption (TSME): %t", platformInfo.TSMEEnabled)
-	writeIndentfln(b, 2, "Author Key ID: %x", report.AuthorKeyEn)
+	writeIndentfln(b, 2, "Author Key ID: %x", report.SignerInfo)
```

Comment:
This line seems to have already been wrong before.
Now, if you want to print the SignerInfo, you should use abi.ParseSignerInfo to get the values.